packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceConfigSet.test.ts

```diff
@@ -5,35 +5,66 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
+import { Config } from '@salesforce/core';
+import { Table } from '@salesforce/salesforcedx-utils-vscode/out/src/output';
 import { expect } from 'chai';
-import { ForceConfigSetExecutor } from '../../../src/commands';
+import * as sinon from 'sinon';
+import { channelService } from '../../../src/channels';
+import { forceConfigSet, ForceConfigSetExecutor } from '../../../src/commands';
 import { nls } from '../../../src/messages';
 
+const CONFIG_KEY = nls.localize('force_config_set_name');
+const CONFIG_TITLE = nls.localize('force_config_set_title');
+
+const sandbox = sinon.createSandbox();
+let channelSpy: sinon.SinonSpy;
+let configSetSpy: sinon.SinonSpy;
+let configWriteSpy: sinon.SinonSpy;
+let tableSpy: sinon.SinonSpy;
+
 describe('Force Config Set', () => {
-  it('should build the force config set command', async () => {
+  beforeEach(() => {
+    channelSpy = sandbox.spy(channelService, 'appendLine');
+    configSetSpy = sandbox.spy(Config.prototype, 'set');
+    configWriteSpy = sandbox.spy(Config.prototype, 'write');
+    tableSpy = sandbox.spy(Table.prototype, 'createTable');
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  it('should set config with the given username or alias', async () => {
     const usernameOrAlias = 'test-username1@gmail.com';
```

Comment:
Small nit: can we move usernameOrAlias to a const at the top of the test file since we're reusing it throughout?